INSTALL.md

```diff
@@ -1,40 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+A helm chart is provided inside the [charts](https://github.com/eclipse-tractusx/sd-factory/tree/main/charts/sdfactory) directory
```

Comment:
I recommend using a relative link and not linking directly to the main branch: [charts](./charts)

Reply:
Should I mention like this ?
A helm chart is provided inside the charts directory in https://github.com/eclipse-tractusx/sd-factory repo

Reply:
Updated with relative link